source/includes/gcp/_target_proxies.md

```diff
@@ -84,13 +89,54 @@ Retrieve a target proxy in an [environment](#administration-environments).
 | -------------------------------- | ------ |
 | `creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format. |
 | `id`<br/>*UUID*                  | The id of the backend service. |
-| `kind`<br/>*string*              | Type of the resource. Either compute#targetHttpProxy, compute#targetHttpsProxy, compute#targetSslProxy or compute#TargetTcpProxy for target proxies.|
+| `kind`<br/>*string*              | Type of the resource. Either compute#targetHttpProxy or compute#targetHttpsProxy for target proxies. |
 | `name`<br/>*string*              | Name of the resource. |
 | `selfLink`<br/>*string*          | Server-defined URL for the resource. |
-| `shortUrlMap`<br/>*string*       | A short version of the URL map. |
-| `type`<br/>*enum*                | A short version of the kind of resource. Either HTTP, HTTPS, SSL or TCP|
+| `shortUrlMap`<br/>*string*       | A short version of the URL map name. |
+| `shortCertificates`<br/>*string* | A short version of the SSL certificate name. |
+| `type`<br/>*enum*                | A short version of the kind of resource. Either HTTP or HTTPS. |
 | `urlMap`<br/>*string*            | URL to the UrlMap resource that defines the mapping from URL to the BackendService. |
 
+<!-------------------- CREATE A TARGET PROXY -------------------->
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/targetProxies"
+
+# Examples:
+```
+
+```json
+{
+   "name": "targetProxyName",
+   "type": "HTTPS",
+   "shortUrlMap": "urlMapName",
+   "shortCertificates": "sslCertificateName"
+}
+```
+
+```json
```

Comment:
duplicate section

Reply:
I want to show two different payloads.

Reply:
Ok maybe add a sentence to both to explain what you are trying to show.

Reply:
How about this? d0eaeeb7df5ae05fb62f6ce9e130749281efd930

Reply:
![image](https://user-images.githubusercontent.com/4197339/66865662-b0e45480-ef65-11e9-90a8-7fe673002915.png)

Reply:
Looks great!